src/main/static/src/components/common/henkilo/HenkiloViewOpenKayttooikeusanomus.js

```diff
@@ -215,7 +221,15 @@ class HenkiloViewOpenKayttooikeusanomus extends React.Component<Props, State> {
         if(this.props.updateHaettuKayttooikeusryhma) {
             this.props.updateHaettuKayttooikeusryhma(id, tila, alkupvm, loppupvm, henkilo, hylkaysperuste || '');
         }
-        this.setState({showHylkaysPopup: false});
+        const disabledMyonnettyButtons = this.state.disabledMyonnettyButtons;
+        const disabledHylkaaButtons = this.state.disabledHylkaaButtons;
```

Comment:
Mutatoidaankohan tässä statea? https://stackoverflow.com/questions/43638938/updating-an-object-with-setstate-in-react